smart-contracts/advanced/multicall.md

```diff
@@ -0,0 +1,55 @@
+---
+description: >-
+  Multicall allows you to aggregate multiple contract reads into a single JSON-RPC request,
+  and execute multiple state-changing calls in a single transaction on the FVM.
+---
+
+# Multicall
+
+Multicall is a powerful tool that offers several advantages for interacting with smart contracts on the Filecoin Virtual Machine (FVM). 
```

Comment:
@xBalbinus can we add more information here to introduce Multicall? I think it will be great to let the readers know that this Multicall is the same one deployed to all other chain. We should also provide links to their website, docs here. For example,

1. Multichain website
2. Multicall docs
3. Multicall contract ABI

Reply:
Multicall docs is the same as the one on GH, website / deployed domains / ABI is linked just now.